app/components/Send/SendAmountsPanel/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+
+import SendAmountsInfoBox from './SendAmountsInfoBox'
+
+import styles from './SendAmountsPanel.scss'
+
+class SendPageHeader extends Component {
+  render() {
+    return (
+      <section className={styles.sendAmountsPanel}>
```

Comment:
Might be cool to have this component scroll horizontally if user has many tokens
<img width="901" alt="screen shot 2018-07-07 at 12 03 52 am" src="https://user-images.githubusercontent.com/13072035/42407410-55f7427c-8179-11e8-822c-d1222ad68380.png">

Reply:
Agree. What would be the best way? Stacked on top of each other and infinite length, or define a max length and set overflow: scroll ?

Reply:
Yes, that would work, I'll implement and we can review if it's clear enough for the user. Perhaps I'll add some sort of indicator when the total amount of tokens exceed the space in the bar. What do you think?